gramps_webapi/api/resources/filters.py

```diff
@@ -74,6 +78,8 @@ def get_custom_filters(args: Dict[str, str], namespace: str) -> List[Dict]:
                     ],
                 }
             )
+    if "filters" in args and len(args["filters"]) != len(filter_list):
+        abort(404)
```

Comment:
Why 404 and not 400, isn't this an invalid request?

Reply:
My thinking is that it did not find all the filters in the list because one was missing, and we would provide 404 for a missing object.  I did the same for rules just above and do the same below in build filter.

Reply:
Ok.